src/mixer/basetrackplayer.h

```diff
@@ -41,6 +42,8 @@ class BaseTrackPlayer : public BasePlayer {
     void loadTrackFailed(TrackPointer pTrack);
     void newTrackLoaded(TrackPointer pLoadedTrack);
     void unloadingTrack(TrackPointer pAboutToBeUnloaded);
+    void noPassthroughInputConfigured();
+    void noVinylControlInputConfigured();
 };
 
 class BaseTrackPlayerImpl : public BaseTrackPlayer {
```

Comment:
Please file a bug to keep track of this renaming TODO as well. (I also prefer clearer naming like Daniel suggests, though would prefer to call one of them (probably Base) just TrackPlayer (since no suffix implies that it's the base class.) )